flink-connectors/flink-connector-cassandra/src/test/java/org/apache/flink/streaming/connectors/cassandra/CassandraSinkBaseTest.java

```diff
@@ -46,15 +47,15 @@
 import static org.assertj.core.api.Assertions.assertThatThrownBy;
 import static org.assertj.core.api.Assertions.fail;
 import static org.mockito.Mockito.mock;
-import static org.powermock.api.mockito.PowerMockito.when;
+import static org.mockito.Mockito.when;
```

Comment:
There are probably 2 major things which from my point of view could be the reason to avoid it
1. PowerMock does not support JUnit5 https://github.com/powermock/powermock/issues/1053
2. PowerMock does not support JDK17 https://github.com/powermock/powermock/issues/1094

Regarding `PowerMockito#when` under the hood it uses the same `Mockito` e.g. https://github.com/powermock/powermock/blob/release/2.x/powermock-api/powermock-api-mockito2/src/main/java/org/powermock/api/mockito/PowerMockito.java#L416-L423